### PR TITLE
add dependabot.yml

### PR DESCRIPTION
**Pull request by Uzlopak**

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

You might want to add fastify's "auto merging if CI pass" action

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "npm"
```
Comment (anonrig) [inline .github/dependabot.yml:9]: Does this work with `pnpm`? I couldn't find a reference in the docs. (https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem)

Comment (anonrig) [reply .github/dependabot.yml:9]: It's not yet supported: https://github.com/dependabot/dependabot-core/issues/1736

Comment (Fdawgs) [reply .github/dependabot.yml:9]: Maybe just use it for GitHub actions for now?

Comment (Fdawgs) [reply .github/dependabot.yml:9]: @Uzlopak

Comment (Uzlopak) [reply .github/dependabot.yml:9]: implement pnpm in dependabot-core

Comment (Uzlopak) [reply .github/dependabot.yml:9]: nevermind. I will not start learning ruby.

I recommend to use npm with package-lock=false

Comment (anonrig) [reply .github/dependabot.yml:9]: Can you remove pnpm lock and start using with package lock false?

**Comment by Uzlopak** [comment]
ugh... so annoying ...

**Pull request closed without merging**